chain/chain/src/store.rs

```diff
@@ -2033,6 +2074,11 @@ impl<'a> ChainStoreUpdate<'a> {
                     &index_to_bytes(height),
                     &epoch_to_hashes,
                 )?;
+                ColBlockPerHeight.gc(
+                    &vec![],
+                    &mut store_update,
+                    Some(&mut self.chain_store.block_hash_per_height),
+                );
```

Comment:
ColBlockPerHeight should not be affected here

Reply:
Sounds good! I think this is a better organized logic too. i did not do that because gc(...) would have to take more arguments that most gc column doesn't need, but it does looks better